engine/src/main/resources/templates/Common/insightsEmailBody.html

```diff
@@ -581,7 +581,7 @@ <h1>{#insert content-title}{/}</h1>
                             <table role="presentation" border="0" align="center" cellpadding="0" cellspacing="0" width="100%">
                                 <tr>
                                     <td class="rh-footer__text">
-                                        This email was sent by Hybrid Cloud Console | <a href="{environment.url}/user-preferences/notifications/{action.bundle}" target="_blank">Manage email preferences</a>
+                                        This email was sent by Hybrid Cloud Console | <a href="{environment.url}/user-preferences/notifications/{action.bundle.or("")}" target="_blank">Manage email preferences</a>
```

Comment:
We don't have bundles in cloud events. With the upcoming change about merging all bundles into a single screen I don't think we need to work on this. Cloud events are still in-progress and it might happen that when we finalize them the bundles are merged anyway.